test/codeu/chat/server/storage/StorageTest.java

```diff
@@ -49,15 +45,71 @@ public void setup(){
 		}
 	}
 
+	//Resets fields for next tests
+	private void reset() throws SQLException{
+		storage.messageTable.destroy();
+		storage.conversationTable.destroy();
+		storage = new Storage(database);
+		conversations.clear();
+	}
+
+	@Before
+	public void setup(){
+		database = new Database("test.db");
+		username = "test";
```

Comment:
The tests are easy to read if you declare the 2 usernames as constants (since they anyway do not change between tests).